ERCS/erc-7969.md

```diff
@@ -0,0 +1,261 @@
+---
+eip: 7969
+title: DKIM Registry Interface
+description: An interface for registering and validating DKIM public key hashes onchain.
+author: Mike Fu (@fumeng00mike), Matthew Yu (@0xknon), Ernesto García (@ernestognw)
+discussions-to: https://ethereum-magicians.org/t/new-erc-dkim-registry-interface/24530
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-06-11
+---
+
+## Abstract
+
+This EIP proposes a standard interface for registering and validating DKIM (DomainKeys Identified Mail) public key hashes on the Ethereum blockchain. The interface allows domain owners to register their DKIM public key hashes and enables third parties to verify the validity of these hashes.
```

Comment:
Not bad, but maybe a tad too brief. You can go into a bit more technical detail (while keeping it an overview). How do DKIM clients get the information, for example?